oat-sa/tao-core
# Fix/add node modules to s3

Upload also `views/node_modules` files to s3

How to test : 
- configure the file `views/build/config/aws.json`
```
cd views/build
npm i
npx grunt  clean:s3 compress:s3 awsS3:upload
```
- check the bucket contains all the files under `node_modules`
